scripts/ci/crates.py

```diff
@@ -459,15 +459,45 @@ def publish(dry_run: bool, token: str) -> None:
         publish_unpublished_crates_in_parallel(crates, version, token)
 
 
-def get_version(finalize: bool, from_git: bool, pre_id: bool) -> None:
-    if from_git:
+def get_latest_published_version(crate_name: str) -> str | None:
+    resp = requests.get(
+        f"https://crates.io/api/v1/crates/{crate_name}",
+        headers={"user-agent": "rerun-publishing-script (rerun.io)"},
+    )
+    body = resp.json()
+
+    if not resp.ok:
+        raise Exception(f"failed to get crate {crate_name}: {body['errors'][0]['detail']}")
+
+    if "versions" not in body:
+        return None
+
+    # response orders versions by semver
+    return body["versions"][0]["num"]
+
+
+class Target(Enum):
+    Git = "git"
+    CratesIo = "cio"
```

Comment:
"cio" is unnecessarily cryptic

```suggestion
    CratesIo = "cratesio"
```